fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -917,6 +917,50 @@ module.exports = function(config, DB) {
         )
 
         test(
+          'openid create and get',
+          function (t) {
+            t.plan(11)
+            var account = {
```

Comment:
Just wondering whether, now that there are three places in this file where we manually create a new account mock, it is worth extracting to a helper function? Lines 15 and 655 are the other places I'm seeing here, but I also have a separate PR that could potentially benefit from it too.